app.js

```diff
@@ -4,7 +4,7 @@ const app = require('./server');
 const applicationStarts = require('./app/lib/prometheus/counters').applicationStarts;
 const log = require('./app/lib/logger');
 
-requireEnv(['API_HOSTNAME', 'API_KEY']);
+requireEnv(['SEARCH_API_KEY']);
```

Comment:
SEARCH_API_HOST and SEARCH_API_VERSION are flagged as required in the README but not included in this check. Is that correct?

Reply:
Yes. Those vars have default values.